### ISSUE TITLE
Build failed. Flutter v 1.12.13+hotfix.5

### ISSUE THREAD

**Issue by malikdoksoz** [Sh1d0w/multi_image_picker#338]
**Bug Report**

`multi_image_picker` version: 4.5.9

Is it happening on Android: yes

Is it happening on iOS: yes

```
 [✓] Flutter (Channel stable, v1.12.13+hotfix.5, on Mac OS X 10.15.2 19C57, locale tr-TR)
    • Flutter version 1.12.13+hotfix.5 at
      /Users/malikdoksoz/TumDosyalar/Dosyalar/flutter_src/flutter
    • Framework revision 27321ebbad (19 hours ago), 2019-12-10 18:15:01 -0800
    • Engine revision 2994f7e1e6
    • Dart version 2.7.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
    • Android SDK at /Users/malikdoksoz/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.0
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3, Build version 11C29
    • CocoaPods version 1.8.3

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 42.1.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.40.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.7.1

[✓] Connected device (1 available)
    • AOSP on IA Emulator • emulator-5554 • android-x86 • Android 9 (API 28) (emulator)

• No issues found!
```

**Describe the bug**

> Compiler message:
../../../../.pub-cache/hosted/pub.dartlang.org/multi_image_picker-4.5.9/lib/src/asset_thumb_provider.dart:31:24: Error: The method 'AssetThumbImageProvider.load' has fewer positional arguments than those of overridden method 'ImageProvider.load'.
  ImageStreamCompleter load(AssetThumbImageProvider key) {

<img width="1128" alt="Ekran Resmi 2019-12-11 23 08 22" src="https://user-images.githubusercontent.com/26676061/70660563-2d946600-1c6b-11ea-9623-65ebea40427c.png">

**Comment by LJjack** [comment]
same question

**Comment by CatEatFishs** [comment]
same question

**Comment by cccchelly** [comment]
change version to 4.6.0-rc.1 ,and i solved it.

**Comment by CatEatFishs** [comment]
I doesn,t solved it

**Comment by Sh1d0w** [comment]
This is fixed in version 4.6.0
